Header header

float64[] audio_feature
float64 audio_power
float64 audio_azimuth
float64[] audio_head_joints
string audio_cmd

float64[] kinematics_ee_pos
float64[] kinematics_ee_quat
float64[] kinematics_jnt_pos
float64[] kinematics_jnt_vel
float64[] kinematics_jnt_eff
float64[] kinematics_target_pos
float64[] kinematics_target_quat

float64[] ft_force
float64[] ft_torque

float64[] vision_pos
float64[] vision_quat

float64[] pps_skin_left
float64[] pps_skin_right


